lib/Tooltip/Tooltip.js

```diff
@@ -156,7 +156,16 @@ module.exports = kind(
 		* @default null
 		* @public
 		*/
-		contentUpperCase: null
+		contentUpperCase: null,
+
+		/**
+		* The property is introduced to support multiline tooltip
+		* when state is declared, it will create a multiline popup.  
+		* @type {string}
+		* @default undefined
+		* @public
+		*/
+		state: undefined
```

Comment:
Ok, thanks for explaining this, I think some of the confusion from the word state, is that it is really 2nd line content. And the word `state` just doesn't lend itself to being verbose that this is content. Seems like if we need to sent this content with a property, we should call it something like `subText`or something to indicate that it is both content AND moves to the next line.

Curious why this couldn't be accomplished using HTML spans inside the tooltip content?